<launch>
  <node pkg="puppeteer_control" type="multi_coordinator" name="coordinator_node"
  	output="screen" required="true" launch-prefix="valgrind --leak-check=full"/>
</launch>